src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -61,19 +61,12 @@ class IndexManagerTests extends SparkFunSuite with SparkInvolvedSuite {
     super.afterAll()
   }
 
-  test("Verify that indexes() returns the correct dataframe.") {
-    import spark.implicits._
-    hyperspace.createIndex(df, indexConfig1)
-    val actual = hyperspace.indexes.as[IndexSummary].collect()(0)
-    val expected = new IndexSummary(
-      indexConfig1.indexName,
-      indexConfig1.indexedColumns,
-      indexConfig1.includedColumns,
-      200,
-      StructType(Seq(StructField("RGUID", StringType), StructField("Date", StringType))).json,
-      s"$indexStorageLocation/index1/v__=0",
-      Constants.States.ACTIVE)
-    assert(actual.equals(expected))
+  test("Verify that indexes() returns the correct dataframe without lineage.") {
```

Comment:
How about:
```scala
test("Verify that indexes() returns the correct dataframe with/without lineage.") {
  Seq("true", "false").foreach { enableLineage =>
    withSQLConf((IndexConstants.INDEX_LINEAGE_ENABLED -> enableLineage) {
      import spark.implicits._
      hyperspace.createIndex(df, indexConfig1)
      ..
    }
  }
}
```

Reply:
Modified all lineage tests to use SqlHelper.